styles/06-components/snackbar.css

```diff
@@ -0,0 +1,39 @@
+.snackbar{
+  display: flex;
+  align-items: center;
+  padding: 0 10px;
+  position: fixed;
+  bottom: 2%;
+  left: 50%;
+  transform: translateX(-50%);
+  border-radius: 10px;
+  font-weight: 200;
+  z-index: 99999;
```

Comment:
why such a large value? i think smaller values would do too

Reply:
to prevent any chance of override